REQUIREMENTS.rst

```diff
@@ -0,0 +1,72 @@
+============================
+ CESM software requirements
+============================
+
+Software requirements for installing, building and running CESM
+===============================================================
+
+Installing, building and running CESM requires:
+
+* a Unix-like operating system (Linux, AIX, OS X, etc.)
+
+* git client version 1.8 or newer
+
+* subversion client version ??? or newer
+
+* python2 version 2.7 or newer
+
+* perl version ??? or newer
```

Comment:
Anyone know what version of perl we need?

Reply:
perl 5.xx